V1/Interfaces/Services/ICryptoService.cs

```diff
@@ -4,8 +4,8 @@ public interface ICryptoService
     {
         string GenerateEncryptedSecretKey();
 
-        void EncryptAsServer<T>(T obj);
-        void DecryptAsServer<T>(T obj);
+        void EncryptAsServer<T>(T obj, bool mobile = false);
```

Comment:
mobile doesn't really make sense in an crypto service. Rather call it useStaticKey or isStatic, meaning it will be a static en- and decryption